seed/data_importer/views.py

```diff
@@ -4,15 +4,16 @@
 :copyright (c) 2014 - 2018, The Regents of the University of California, through Lawrence Berkeley National Laboratory (subject to receipt of any required approvals from the U.S. Department of Energy) and contributors. All rights reserved.  # NOQA
 :author
 """
-import csv
 import base64
+import csv
+import datetime
 import hashlib
 import hmac
 import json
 import logging
 import os
-import pint
 
+import pint
```

Comment:
Import changes are just reordering them and then adding the datetime import to allow us to generate a date stamp